modules/get-started/pages/whats-new.adoc

```diff
@@ -6,6 +6,51 @@ This topic includes new content added from version 4.29.0 onwards.
+
+== Version 4.41.0
+
+This release adds the `max_records_per_request` field to the xref:components:outputs/aws_sqs.adoc[`aws_sqs` output].
+
+=== Packaging update
+
+Starting from version 4.41.0, the `-cgo` suffixed Docker image is no longer available due to low demand and the unacceptable cadence with which the image base (Debian) receives security updates.
+
+To create your own CGO builds, run the following command:
+
+```bash
+CGO_ENABLED=1 make TAGS=x_benthos_extra redpanda-connect. 
+```
+
+== Version 4.40.0
+
+This update includes a beta release of the xref:components:inputs/pg_stream.adoc[`pg_stream` input] for Self-Managed and Cloud, which captures data changes made to a PostgreSQL database in real time and streams them to Redpanda Connect.
+
+=== Component updates
+
+This release adds:
+
+* The `metadata_max_age` field to the xref:components:outputs/redpanda_migrator_offsets.adoc[`redpanda_migrator_offsets` output].
+* The `timestamp_ms` field to the following outputs: xref:components:outputs/ockam_kafka.adoc[`ockam_kafka`], xref:components:outputs/kafka.adoc[`kafka`], xref:components:outputs/kafka_franz.adoc[`kafka_franz`], xref:components:outputs/redpanda.adoc[`redpanda`], xref:components:outputs/redpanda_common.adoc[`redpanda_common`], xref:components:outputs/redpanda_migrator.adoc[`redpanda_migrator`].
+* The `kafka_timestamp_ms` metadata field, which uses a Unix timestamp with millisecond precision, to the following inputs: xref:components:inputs/kafka.adoc[`kafka`], xref:components:inputs/kafka_franz.adoc[`kafka_franz`], xref:components:inputs/redpanda.adoc[`redpanda`], xref:components:inputs/redpanda_common.adoc[`redpanda_common`], xref:components:inputs/redpanda_migrator.adoc[`redpanda_migrator`].
+* The `translate_schema_ids` and `schema_registry_output_resource` to the xref:components:outputs/redpanda_migrator.adoc[`redpanda_migrator` output].
+* The `backfill_dependencies` and `input_resource` fields to the xref:components:outputs/schema_registry.adoc[`schema_registry` output].
+* Options to optimize the build of the data output from the xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output].
+
+Other changes:
+
+* The following components now use the https://github.com/twmb/franz-go/tree/master/pkg/sr[Franz Kafka Schema Registry client library^]: xref:components:inputs/schema_registry.adoc[`schema_registry` input] and xref:components:outputs/schema_registry.adoc[output], and the xref:components:processors/schema_registry_encode.adoc[`schema_registry_encode`] and xref:components:processors/schema_registry_decode.adoc[`schema_registry_decode`] processors.
+* The xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output] can now automatically create tables in Snowflake, when the `schema_evolution.enabled` field is set to `true`.
+
+==== Deprecations:
+
+This release removes: 
```

Comment:
It'd be useful to make it easy to scan for affected components and the new name. Alos, we should call out any actions users must take. Maybe something like:

Inputs

**Deprecated field**: `kafka_timestamp`
**Affected components**:
- redpanda`
- ...
**Replaced by**: `kafka_timestamp_ms`

Action required

Modify all systems that process these fields to ensure compatibility with the new field names.